doc/submit_obs.md

```diff
@@ -0,0 +1,21 @@
+# Submitting observations
+
+This page describes the protocol and parameters necessary for submitting new
+observations. It is a work in progress.
+
+## Configuration
+
+Configuration should be submitted as JSON. There's a number of parameters that
+are (almost) mandatory:
```

Comment:
Maybe we should print the warnings when the "almost" mandatory field is missing?

Reply:
Added as suggested.